ironpython-git/PKGBUILD

```diff
@@ -1,19 +1,17 @@
 # Maintainer: Michael Schubert <mschu.dev at gmail>
 
 pkgname=ironpython-git
-pkgver=2.7.2.a2.471.g7be8b73
+pkgver=ac72964d7
```

Comment:
Then we should probably change pkgver function so it updates this variable correctly.